ElectronClient/style.css

```diff
@@ -200,3 +200,22 @@ a {
 .cm-header-1, .cm-header-2, .cm-header-3, .cm-header-4, .cm-header-5, .cm-header-6 {
 	line-height: 1.5em;
 }
+
+.mark {
+	background: #F3B717;
+	color: black;
+}
+
+.mark-selected {
+	background: #CF3F00;
+	color: white;
+}
+
+.mark-scrollbar {
+	background: #F3B717;
+  border-top: 1px solid orange;
+  border-bottom: 1px solid orange;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  opacity: .5;
+}
```

Comment:
Ideally all CSS should be part of the component, like it's done for TinyMCE.tsx for example. All the colours in particular should come from theme whenever possible (preferably by re-using an existing colour).

Reply:
I was wondering about the best way to do this, I should have thought to check TinyMCE.

I've moved all the codemirror specific stuff out of style.css and into CodeMirror.tsx. I still have the theme imports in index.html, but I plan to address that in another pull request.